.chloggen/fix_preserve-metric-names-request-split.yml

```
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: exporterhelper
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fixes an issue in exporterhelper's sending_queue where metric names were lost during metricRequest splitting.
```

exporterhelper is already specified as the component name. 

```suggestion
note: Preserve all metrics metadata when batch splitting.
```